public/manifest.json

```diff
@@ -19,7 +20,42 @@
     }
   ],
   "start_url": "./",
-  "display": "standalone",
+  "display": "fullscreen",
   "theme_color": "#000000",
-  "background_color": "#222222"
-}
+  "background_color": "#222222",
+  "screenshots": [
+    {
+      "src": "logo192.png",
+      "sizes": "192x192",
+      "type": "image/png"
+    }
+  ],
```

Comment:
I think that having a `screenshots` property is a great idea, but we should probably use images of Chitchatter in use rather than just the logo. I can make a few when I have time, or you can make them if you'd like and can get to it first.

Reply:
"display": "standalone",
 "display": "fullscreen",

fullscreen - it will work on mobile view and for desktop view the standalone and fullscreen are same

the fullscreen display will help the user to nevigate their destination and there will be no wastage on the whole screen of the phone 

## Example - when fullscreen

![Screen Shot 2022-12-11 at 16 13 20](https://user-images.githubusercontent.com/85288261/206907544-94b4ba3f-0373-470e-b70b-37a73a4eb73b.png)

## Example - when standalone

![image](https://user-images.githubusercontent.com/85288261/206907833-3f43a7cb-fdc6-4321-acf9-c96c4b307459.png)

Reply:
(It looks like the above comment was meant for [the other threat](https://github.com/jeremyckahn/chitchatter/pull/80/files#r1042432575).)